syntax = "proto3";
package user;

message GetUserRequest { int32 id = 1; }
message UserResponse {
  int32 id = 1;
  string name = 2;
  string email = 3;
}

service UserService {
  rpc GetUser(GetUserRequest) returns (UserResponse);
}
